_includes/layout/base/nav.html

```diff
@@ -0,0 +1,32 @@
+<nav class="navbar">
+  <div class="container">
+    <div class="navbar-header">
+      <a class="navbar-brand clearfix" href="{{ site.baseurl }}/">
+        <img src="{{ site.baseurl }}/images/trans/5.png" alt="Forseti Security Logo" />
+      </a>
```

Comment:
nit: I know I removed the title [here](https://github.com/denzel-morris/forseti-security/commit/c3cc860663c36e594322833627a34e8349f4bf5d#diff-70c103dc160c67e7a579206a58d26b94), but can you add it back, it's better there when you're not on the home page.